src/Microsoft.NetCore.Analyzers/Core/xlf/MicrosoftNetCoreAnalyzersResources.cs.xlf

```diff
@@ -68,13 +68,28 @@
         <note />
       </trans-unit>
       <trans-unit id="DoNotUseInsecureRandomnessDescription">
+        <source>Using a cryptographically weak pseudo-random number generator may allow an attacker to predict what security sensitive value will be generated. Use a cryptographically strong random number generator if an unpredictable value is required, or ensure that weak pseudo-random numbers aren't used in a security sensitive manner.</source>
+        <target state="new">Using a cryptographically weak pseudo-random number generator may allow an attacker to predict what security sensitive value will be generated. Use a cryptographically strong random number generator if an unpredictable value is required, or ensure that weak pseudo-random numbers aren't used in a security sensitive manner.</target>
```

Comment:
><target state="new"> [](start = 8, length = 20)

Should probably manually swap DoNotUseInsecureRandomnessDescription / Message so that localizers don't have to retranslate the strings (keep state="translated")

Reply:
You also have to keep the translated text from before your change.

---
In reply to: [329818334](https://github.com/dotnet/roslyn-analyzers/pull/2822#discussion_r329818334) [](ancestors = 329818334)